components/form/Label.js

```diff
@@ -13,3 +14,14 @@ export function Label({ floatLabel, className, name, htmlFor, text }) {
     </label>
   );
 }
+
+Label.propTypes = {
```

Comment:
We need to also define `floatLabel` and `text` here and defaultProps (if necessary)